components/layout_2020/flexbox/layout.rs

```diff
@@ -954,6 +956,11 @@ impl FlexLine<'_> {
                 //  it’s a max violation.”
                 for (item_and_target_main_size, frozen) in items() {
                     if violation(item_and_target_main_size) < Length::zero() {
+                        let (item, target_main_size) = item_and_target_main_size;
+                        let Some(max_size) = item.content_max_size.main else {
+                            unreachable!("")
```

Comment:
I cant' find other uses of `unreachable!("")`. You should probably use `unreachable!()` or provide an actual message.

But I don't really see the advantage over the previous `unwrap()`.

Reply:
Because using `unwrap()` is unholy :)

Reply:
Updated.